src/contracts/tokens/ProtocolToken.sol

```diff
@@ -38,9 +42,14 @@ contract ProtocolToken is ERC20, ERC20Permit, ERC20Votes, Authorizable, IProtoco
     _burn(msg.sender, _wad);
   }
 
+  /// @inheritdoc IProtocolToken
+  function unpause() external isAuthorized {
+    _unpause();
+  }
+
   // --- Overrides ---
 
-  function _update(address _from, address _to, uint256 _value) internal override(ERC20, ERC20Votes) {
+  function _update(address _from, address _to, uint256 _value) internal override(ERC20, ERC20Votes, ERC20Pausable) {
     super._update(_from, _to, _value);
```

Comment:
```suggestion
    ERC20Votes._update(_from, _to, _value);
```

Reply:
Then there's no reason to inherit `ERC20Pausable`, but `Pausable`.

Reply:
OHHHHH SHI, they changed the implementation to this sh*t! 😡 💢

Reply:
wdyt about:
```suggestion
    if (_from != 0) ERC20Pausable._update(_from, _to, _value);
    else ERC20._update(_from, _to, _value);
```

i think that with current OZ setup, at SOME point we're gonna have to be making sth like this, since there's no `_beforeTokenTransfer` hook anymor